root/.one_params_template

```diff
@@ -0,0 +1,11 @@
+{ 
+    "ALYX_LOGIN": "ALYX_USERNAME",
+    "ALYX_PWD": "ALYX_PASSWORD",
+    "ALYX_URL": "https://alyx.internationalbrainlab.org",
+    "CACHE_DIR": "/src/ibl-pipeline/data/FlatIron",
+    "FTP_DATA_SERVER": "ftp://ibl.flatironinstitute.com",
+    "FTP_DATA_SERVER_LOGIN": "iblftp",
```

Comment:
Is this the real login? If so, should it be changed to a placeholder like `ALYX_USERNAME` above?